pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -824,9 +833,9 @@ monitorLoop:
 		default:
 		}
 
-		stats, err := dom.GetJobInfo()
+		stats, err := dom.GetJobStats(0)
```

Comment:
we moved to using GetJobStats now, but this code has been refactored. Take a look at the new `pkg/virt-launcher/virtwrap/live-migration-source.go` file.